pkg/test/helpers.go

```diff
@@ -17,7 +17,8 @@ import (
 // package of the caller and unmarshals the contents of that file into `obj`.
 // Any errors will fail the `t`est.
 func LoadYAML(t *testing.T, obj interface{}, filename string) {
-	bytes, err := ioutil.ReadFile(filepath.Join("testdata", filename))
+	filename = filepath.Join("testdata", filename)
+	bytes, err := ioutil.ReadFile((filename))
```

Comment:
Why this change?
...but assuming it's necessary, might as well collapse the redundant parens:
```suggestion
	filename = filepath.Join("testdata", filename)
	bytes, err := ioutil.ReadFile(filename)
```